bin/index

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+for f in $@; do echo $f; done | parallel ./bin/ts-earch | jq -c '.[]' | jq --slurp -c
```

Comment:
```suggestion
# Usage:
#
# $ ./bin/index ~/path/to/directories/* > index.json`
for f in $@; do echo $f; done | parallel ./bin/ts-earch | jq -c '.[]' | jq --slurp '.'
```

- Removed `-c` to get a formatted output.
- Added a filter (`'.'`) otherwise redirecting output to file didn't work